packages/react-server/core/ClientController.js

```diff
@@ -55,6 +55,7 @@ class ClientController extends EventEmitter {
 
 		window.__reactServerTimingStart = window.performance ? window.performance.timing.navigationStart : new Date;
 		var wakeTime = new Date - window.__reactServerTimingStart;
+		var aboveTheFold = window.__displayAboveTheFold - window.__reactServerTimingStart;
```

Comment:
So, this could be negative if `!window.performance`?  Should we handle that case?